docs/editor/artificial-intelligence.md

```diff
@@ -87,11 +79,82 @@ In the image above, Copilot presents three suggestions. You can accept the entir
 
 If you don't want to accept any of the suggestions, you can continue typing, and Copilot will continue providing suggestions as you work.
 
+## Getting the most out of Copilot
+
+There are things you can do to help ("prompt") Copilot give you the best possible suggestions. And the good news is that you are probably already doing right things since they help you and your colleagues understand your code.
```

Comment:
> And the good news is that you are probably already doing right things since they help you and your colleagues understand your code.


I'm not sure I understand this line. Are we saying folks may already be doing prompt engineering?

Reply:
Folks should already be doing things like:

Putting a high level comment at the top of a file
Be using good function names
Be using good function comments to help explain the code

Doing the things that aid code maintenance and help anyone (human or AI) understand the code.

My take on actual prompt engineering is that it is the work to take the available context and package it up (what to include or not) for the AI query. The user sets the available context and the Copilot extension (client side) creates the prompt.

I'll try to make this clearer.